src/kbc_runtime.rs

```diff
@@ -54,6 +54,23 @@ impl KbcRuntime {
         Ok(plain_payload)
     }
 
+    pub fn get_resource(
+        &mut self,
+        kbc_name: String,
+        kbs_uri: String,
+        resource_description: String,
+    ) -> Result<Vec<u8>> {
+        if self.kbc_instance_map.get_mut(&kbc_name).is_none() {
+            self.instantiate_kbc(kbc_name.clone(), kbs_uri)?;
+        }
+        let kbc_instance = self
+            .kbc_instance_map
+            .get_mut(&kbc_name)
+            .ok_or(anyhow!("KBC runtime: The KBC instance does not existing!"))?;
+        let resource = kbc_instance.get_resource(resource_description)?;
```

Comment:
How can we handle KBCs that don't have a get_resource endpoint? I think we might want to add that endpoint as a stub to every existing KBC and create some mechanism for reporting that the feature isn't supported.

Reply:
In the trait definition of KBC interface, we can add the default implementation of KBC's get resource interface: https://github.com/confidential-containers/attestation-agent/pull/43/commits/4f3ae318b4fde38b637ac38bb41992987900177a 

For KBC without custom implementation of get resource interface, this default implementation will be used.